internal/worker_runner.go

```diff
@@ -66,7 +65,7 @@ func NewWorkerRunner(addr *common.Address, loggerSettingsSvc loggers.SettingsSto
 }
 
 // Max failures allowed for a PID before sending an error to the cloud
-const maxPidFailures = 10
+const maxPidFailures = 30
```

Comment:
@JamesReate , set to 30, or we want to get rid from logging it at all?

Reply:
or we can remove above send to mqtt and leave only https://github.com/DIMO-Network/edge-network/blob/776b5af2b1f130ac8ed23a13a189d82272baaf57/internal/worker_runner.go#L182

Reply:
not send to mqtt